### PR TITLE
Add compat data for File

### PR DESCRIPTION
**Pull request by CalmBit**
Hey all,

Here's another one. There's a lot of proprietary, obsolete methods and properties for this object, which seem to lack compatibility tables to translate. Should I ignore these?

### CODE REVIEW

File: api/File.json [hunk 1]
```diff
@@ -0,0 +1,307 @@
+{
+  "api": {
+    "File": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/File",
+        "support": {
+          "webview_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/File.json]: Should be `true`.

File: api/File.json [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+          },
+          "chrome": {
+            "version_added": "13"
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`

File: api/File.json [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+          "deprecated": false
+        }
+      },
+      "File": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/File",
+          "description": "<code>File()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "4.4.4"
```
Comment (jpmedley) [inline api/File.json]: Change to `true`. Webview has parity with Chrome versions, not Android versions.

File: api/File.json [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+            },
+            "chrome": {
+              "version_added": "13"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`.

File: api/File.json [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "lastModified": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/lastModified",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/File.json]: `true`.

File: api/File.json [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+            },
+            "chrome": {
+              "version_added": "13"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`.

File: api/File.json [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "lastModifiedDate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/lastModifiedDate",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/File.json]: `true`.

File: api/File.json [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+            },
+            "chrome": {
+              "version_added": "13"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`

File: api/File.json [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+            "deprecated": true
+          }
+        }
+      },
+      "name": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/name",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/File.json]: `true`

File: api/File.json [hunk 10]
```diff
@@ -0,0 +1,307 @@
...
+            },
+            "chrome": {
+              "version_added": "13"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`.

File: api/File.json [hunk 11]
```diff
@@ -0,0 +1,307 @@
...
+            "deprecated": false
+          }
+        }
+      },
+      "type": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/File/type",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/File.json]: `true`

File: api/File.json [hunk 12]
```diff
@@ -0,0 +1,307 @@
...
+            },
+            "chrome": {
+              "version_added": "13"
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/File.json]: `true`

**Comment by CalmBit** [comment]
Ok, awesome! Fixed those right up for you :smile:

**Pull request merged**